scripts/cybershake/queue_monitor.py

```diff
@@ -151,6 +153,49 @@ def update_tasks(
                     "Disappeared from squeue. Creating a new task.",
                 )
             )
+            # When job failed, we want to log metadata as well
+            # Assumes that when enter a jobid, only one row will be will returnerd
+            # Sometimes, when enter a job id eg.578928, several rows would be returned eg.
+            # 578928         u-bl689.atmos+    01:45:32  65-09:41:09  1840          COMPLETED
+            # 578928.batch   batch             01:45:32    00:32.704    40  458464K COMPLETED
+            # 578928.extern  extern            01:45:32     00:00:00  1840    1368K COMPLETED
+            # 578928.0       rose-mpi-laun+    01:45:10  65-09:40:37   896 1799220K COMPLETED
+
+            run_time = (
+                subprocess.check_output(
+                    "sacct -n -j {} -A nesi00213 --format=elapsed".format(
+                        db_running_task.job_id
+                    ),
+                    shell=True,
+                )
+                .decode("utf-8")
+                .strip()
+                .split()[0]
+            )
+            # if run_time exceeds one day
+            if "-" in run_time:
```

Comment:
forcing the output to use "CPUTimeRAW"
can force the output to be in seconds, which can then avoid all these manual transformation of datetime format.